src/app.js

```diff
@@ -102,25 +103,65 @@ go.app = function() {
+        };
+
+        sub_section.demographics = function(data) {
+            return "RSA Citizens: " + data.citizenship_south_african.values.this + "%\n" + "Most spoken language: " + data.language_most_spoken.name;
+        };
+
+        sub_section.households = function(data) {
+            return "Informal Dwellings: " + data.informal.values.this + "%\n" + "Homes owned and paid off: " + data.tenure_distribution["Owned and fully paid off"].values.this + "%" ;
+        };
+
+        sub_section.service_delivery = function(data) {
+            return "Electricity: " + data.electricity_access_distribution.total_all_elec.values.this + "%\n" + "No toilet access: " + data.percentage_no_toilet_access.values.this + "%\n" + "Water from service provider: " + data.percentage_water_from_service_provider.values.this + "%";
+        };
+
+        sub_section.economics = function(data) {
+            return "Individual Income:\n" + data.individual_income_distribution.R0.name + ": " + data.individual_income_distribution.R0.values.this + "%\n" + "R13k - R26k: " + data.individual_income_distribution["R13k - R26k"].values.this + "\n Work in formal sector: " + data.sector_type_distribution["In the formal sector"].values.this + "%\n" + "Work in informal sector: " + data.sector_type_distribution["In the informal sector"].values.this + "%\n" + "Discouraged work seeker: " + data.employment_status["Discouraged work-seeker"].values.this + "%\n" + "Employed: " + data.employment_status.Employed.values.this + "%\n" + "Not economically active: " + data.employment_status["Other not economically active"].values.this + "%\n" + "Unemployed: " + data.employment_status.Unemployed.values.this + "%";
+        };
+
+        sub_section.education = function(data) {
+            return "Matric or higher: " + data.educational_attainment.percent_fet_or_higher.values.this + "%\n" + "Grade 9 or higher: " + data.educational_attainment.percent_get_or_higher.values.this + "%\n";
+        };
+
+        sub_section.children = function(data) {
+            return "Children (<18): " + data.demographics.child_adult_distribution['Children (< 18)'].values.this + "%";
+        };
+
+        sub_section.child_households = function(data) {
+            return "Child-headed households in informal dwellings: " + data.child_households.informal.values.this + "%";
+        };
+
         self.states.add('states:display-data', function(name, opts) {
             var section_data = opts.opts_data[opts.section_id]; 
-            var sub_section_data = _.map(section_data, function(d) {
-                return ('Registered Voters: ' + d.registered_voters.values.this);
-            });
-            return new ChoiceState(name, {
+            var return_text = sub_section(section_data, opts.section_id);
+
+            return new PaginatedChoiceState(name, {
```

Comment:
We only have two choices so I'm not sure how useful PaginatedChoiceState will be here. We might want to either use PaginatedState (definitely leave that for another PR) and just use an ordinary ChoiceState here for now.